distributed/tests/test_actor.py

```diff
@@ -290,7 +290,6 @@ async def test_failed_worker(c, s, a, b):
 
     assert "actor" in str(info.value).lower()
```

Comment:
how about moving this all into the pytest raises? `with pytest.raises(ValueError, match=r"Worker holding Actor was lost\.  Status: error'):`